src/SearchInput/SearchInput.tsx

```diff
@@ -41,6 +52,10 @@ type SearchInputProps = {
   onFound: (element: string) => void
   /** Displays search results in a relative position to other elements */
   resultsRelativePosition?: boolean
+  /** Displays caret */
+  showCaret?: boolean
+  /** Displays border */
+  hasBorder?: boolean
```

Comment:
can you call this `outlined` so it's in line with the same prop we already have in other components?